collectd-elasticsearch/docs/counter.thread_pool.rejected.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Rejected Rejected Requests
+brief: Number of rejected requests
+metric_type: counter
+---
+### Rejected Requests
+
+> The number of requests that have been rejected on this node since it started.
+> per thread pool
+
+If the get request queue fills up to its limit, new work units will begin to be
+rejected, and you will see that reflected in this rejected metric. This is 
+often a sign that your cluster is starting to bottleneck on some resources, 
+since a full queue means your node/cluster is processing at maximum speed but 
+unable to keep up with the influx of work.
+
+Number of rejected requests for a given thread pool.  The represented thread pool is indicated by the dimension "thread_pool".
+
+
+The following thread pools may be configured:
+| Thread Pool | Description | Configured By Default |
+| ----------- | ----------- | --------------------- |
```

Comment:
Spaces between | and - cause this not to render as a table. Check out the rendered view.